### ISSUE TITLE
LWFA simulation

### ISSUE THREAD

**Issue by Toader22** [SmileiPIC/Smilei#128]
Dear developers,  

I need to study the laser wakefield acceleration of the electrons in the bubble regime at the interaction of the ultra-high power laser pulse with a Helium gas with the density 0.05Nc (Nc is the critical density of the plasma - the density of the plasma for which the plasma frequency is equal with the laser frequency) and the intensity of the laser pulse 10^22 W/cm^2. The Helium gas target has the thickness of 111um. Simulation time must be 2 ps. The grid step size in the x direction and y direction is 80nm. The timestep of the simulation is 0.26 fs. In a grid cell must be around 21 macroparticles. 

Basically: 
a high intensity laser pulse with
the wavelenght 800nm 
the period 2.66 fs 
duration 25 fs
peak energy of 25 J 
(gaussian laser pulse and liniar p-polarized, i.e Ey=/=0 and Ez=0) 
The laser pulse comes from the left side of the target and there is a vaccum before the plasma. 

particles per cell = 5
vacuum = 50 
for laser: a0 = 68 
Ne = 0.05 Nc 
nx = 1300 
ny = 900 
dx = 2

Can you please help me with the code for this simulation? I tried to modify the benchmarks that are provided here, but with no success.. :( 

Thank you very much for your time :)

**Comment by mccoys** [comment]
Hello,

Have you read carefully the ` units` section of the documentation?  Have you tried the examples of laser wake in the benchmarks folder?  Have you followed the LWFA examples in the tutorials?

**Comment by Toader22** [comment]
I tried, but I don.t know how to to write the code for the simulation with those parameters. Could you please help me in this matter? 
![simulation](https://user-images.githubusercontent.com/52110467/60021330-7abf5400-969a-11e9-8292-a32817378576.jpeg)

**Comment by mccoys** [comment]
Please be more specific on what you have tried and why you think it does not work. You must understand we are not doing the work for you. It is your task to learn how to use our code. Note that there is a lot of documentation and help online. We can help if the documentation is not clear, or if something needs more explanation.

**Comment by lesnat** [comment]
Is this input file for the Smilei code ?
It very looks like PICLS input file.
